presto-native-execution/presto_cpp/main/common/Configs.h

```diff
@@ -313,6 +313,12 @@ class SystemConfig : public ConfigBase {
   /// Ignored if zero. Default is zero.
   static constexpr std::string_view kWorkerOverloadedThresholdCpuPct{
       "worker-overloaded-threshold-cpu-pct"};
+  /// Floating point number used in calculating how many drivers must be queued
+  /// for the worker to be considered overloaded.
+  /// Ignored if zero. Default is zero.
+  static constexpr std::string_view
+      kWorkerOverloadedThresholdNumQueuedDriversHwMultiplier{
```

Comment:
Can we make the name more concise and shorter such that it is easier to read?

Reply:
I'm not a fan of this either.
However, we need all the words in this monstrous name, taking one or two out would undermine its meaning.